### PR TITLE
AAD env var rename, enable AAD tests

### PR DESCRIPTION
**Pull request by timtay-microsoft**
.NET SDK will be using IOTHUB_CLIENT_ID, IOTHUB_CLIENT_SECRET, and MSFT_TENANT_ID as the environment variable names as well.

### CODE REVIEW

File: iot-e2e-tests/android/app/build.gradle [hunk 1]
```diff
@@ -14,7 +14,7 @@ def IS_BASIC_TIER_HUB = project.hasProperty('IS_BASIC_TIER_HUB') ? '"'+project.p
 def IS_PULL_REQUEST = project.hasProperty('IS_PULL_REQUEST') ? '"'+project.property('IS_PULL_REQUEST')+'"' : '"false"'
 def IOTHUB_CLIENT_ID = project.hasProperty('IOTHUB_CLIENT_ID') ? '"'+project.property('IOTHUB_CLIENT_ID')+'"' : '""'
 def IOTHUB_CLIENT_SECRET = project.hasProperty('IOTHUB_CLIENT_SECRET') ? '"'+project.property('IOTHUB_CLIENT_SECRET')+'"' : '""'
```
Comment (timtay-microsoft) [inline iot-e2e-tests/android/app/build.gradle:16]: @vinagesh it turns out I already had the "IOTHUB_" prefix for these environment variables, so I just needed to change the TENANT_ID to be prefixed with MSFT

File: iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java [hunk 2]
```diff
@@ -253,29 +253,6 @@ public void getDigitalTwinWithAzureSasCredential() {
         assertEquals(responseWithHeaders.body().getMetadata().getModelId(), E2ETestConstants.THERMOSTAT_MODEL_ID);
     }
 
-    @Ignore("RBAC authentication isn't supported on hub yet")
```
Comment (timtay-microsoft) [inline iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java:256]: As mentioned in the description of this PR, the service won't support PnP APIs yet, so the SDK has been stripped of it's TokenCredential digital twin APIs

File: vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml [hunk 3]
```diff
@@ -180,7 +180,7 @@ jobs:
           IS_BASIC_TIER_HUB: $(IS-BASIC-TIER-HUB)
           IOTHUB_CLIENT_SECRET: $(IOTHUB-CLIENT-SECRET)
           IOTHUB_CLIENT_ID: $(IOTHUB-CLIENT-ID)
-          IOTHUB_TENANT_ID: $(IOTHUB-TENANT-ID)
```
Comment (vinagesh) [inline vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml:183]: Can you check if java uses this anywhere? LA_AAD_TENANT
If yes, I'm changing the script to use the same MSFT_TENANT_ID. So you might want to change that.

**Review by vinagesh — APPROVED** [review]

File: vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml (hunk 3 above)
Comment (timtay-microsoft) [reply vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml:183]: Nope, no references found for "LA_AAD_TENANT" in this repo

**Pull request merged**